### PR TITLE
Documentation tool: Add missing error messages

### PR DESCRIPTION
**Pull request by maxGimeno**
## Summary of Changes

Add missing error text in cases the build job returns 1.

### CODE REVIEW

**Comment by lrineau** [comment]
As far as I understand, there is no need to test this PR in `integration`: only the Github Actions workflows are affected.

**Comment by maxGimeno** [comment]
Yes but it is not finished

**Comment by maxGimeno** [comment]
Ok for integration

File: .github/workflows/build_doc.yml [hunk 1]
```diff
@@ -102,7 +110,7 @@ jobs:
             mv tmp.html index.html
             git add ${PR_NUMBER}/$ROUND index.html && git commit -q --amend -m "base commit" && git push -q -f -u origin master
           else
-            echo "::set-output name=DoxygenError::This round already exists."
+            echo "::set-output name=DoxygenError::This round already exists. Overwrite it with `/build-force`"
```
Comment (lrineau) [inline .github/workflows/build_doc.yml:113]:
```suggestion
            echo "::set-output name=DoxygenError::This round already exists. Overwrite it with `/force-build`"
```

**Comment by lrineau** [comment]
Oops. This PR was marked for 5.2.4 but was based on `master`.

... Anyway, `/build` will probably always be in a PR based on `master`.

**Pull request merged**